src/command_line/search/basic_search.rs

```diff
@@ -1,8 +1,8 @@
-pub use super::prelude::*;
+pub use super::super::prelude::*;
```

Comment:
I wonder if we should make `use` statements relative to `crate`? Mixing and matching relative vs absolute could be confusing.